src/datashare/azext_datashare/generated/_help.py

```diff
@@ -0,0 +1,205 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=line-too-long
+# pylint: disable=too-many-lines
+
+from knack.help_files import helps
+
+
+helps['datashare account'] = """
+    type: group
+    short-summary: datashare account
+"""
+
+helps['datashare account list'] = """
+    type: command
+    short-summary: List Accounts in Subscription
+    examples:
+      - name: Accounts_ListByResourceGroup
+        text: |-
+               az datashare account list --resource-group "SampleResourceGroup"
+"""
+
+helps['datashare account show'] = """
+    type: command
+    short-summary: Get an account
+    examples:
+      - name: Accounts_Get
+        text: |-
+               az datashare account show --account-name "Account1" --resource-group
+               "SampleResourceGroup"
+"""
+
+helps['datashare account create'] = """
+    type: command
+    short-summary: Create an account
+    examples:
+      - name: Accounts_Create
+        text: |-
+               az datashare account create --identity type=SystemAssigned --location "West US 2" --tags
+               tag1=Red tag2=White --account-name "Account1" --resource-group "SampleResourceGroup"
+"""
+
+helps['datashare account update'] = """
+    type: command
+    short-summary: Patch an account

```

Comment:
Update an account. pls feedback to codegen team

Reply:
The message is from swagger, modified it in manual code.